ugvc/mrd/srsnv_training_utils.py

```diff
@@ -483,11 +493,30 @@ def save_model_and_data(self):
             "train_set_size": self.train_set_size,
             "test_set_size": self.test_set_size,
             "fp_featuremap_entry_number": self.fp_featuremap_entry_number,
+            "lod_filters": self.lod_filters,
+            "adapter_version": self.adapter_version,
+            "columns": self.columns,
         }
 
         with open(self.params_save_path, "w", encoding="utf-8") as f:
             json.dump(params_to_save, f)
 
```

Comment:
TODO: merge the two reports so train and test set results are presented together

Reply:
TODO: change the report so it produces metrics that are saved into a h5 table in addition to the plots

Reply:
added